crates/anvil-polkadot/src/cmd.rs

```diff
@@ -245,6 +250,26 @@ pub struct AnvilEvmArgs {
     pub memory_limit: Option<u64>,
 }
 
+#[derive(Clone, Debug, Parser)]
+#[command(next_help_heading = "Fork options")]
+pub struct ForkArgs {
```

Comment:
why are we not using the forking arguments as defined in anvil's AnvilEvmArgs?
We should also use the same names so that we provide compatibility and a seamless experience